themes/bootstrap3/templates/header.phtml

```diff
@@ -65,33 +65,39 @@
           <?php endif; ?>
 
           <?php if (isset($this->layout()->themeOptions) && count($this->layout()->themeOptions) > 1): ?>
-            <li class="theme dropdown">
-              <a href="#" class="dropdown-toggle" data-toggle="dropdown"><?=$this->transEsc("Theme")?><?=$this->icon('dropdown-caret') ?></a>
-              <ul class="dropdown-menu">
-                <?php foreach ($this->layout()->themeOptions as $current): ?>
-                  <li<?=$current['selected'] ? ' class="active"' : ''?>>
-                    <a href="<?=$this->escapeHtmlAttr($this->url()->addQueryParameters(['ui' => $current['name']])) ?>" rel="nofollow">
-                      <?=$this->transEsc($current['desc']) ?>
-                    </a>
-                  </li>
-                <?php endforeach; ?>
-              </ul>
-            </li>
+              <?=$this->component('menu-button', [
+                  'toggleLabel' => 'Theme',
+                  'wrapperTag' => 'li',
+                  'menuItems' => array_map(
+                      function ($current) {
+                          $item = $current;
```

Comment:
Do we need this assignment? Can we just call the parameter $item? (I don't think this is getting passed by reference, so I don't think the assignment matters).

Reply:
Sorry I spent a lot of time writing code where the linter yelled at me if I changed parameters at all. Fixed!